projects/kit/components/tag/tag.style.less

```diff
@@ -170,17 +170,16 @@
     }
 }
 
-.text {
-    .text-overflow();
+.t-text {
     padding-right: 0.02em;
 }
 
-.tag {
+.t-tag {
+    .text-overflow();
     position: relative;
     display: flex;
     align-items: center;
     border-radius: inherit;
-    white-space: pre;
```

Comment:
I wonder if it was there for a reason 🤔

Reply:
I have a hard time understanding safari (((